crates/nu-lsp/src/notification.rs

```diff
@@ -23,12 +21,6 @@ impl LanguageServer {
                         .expect("Expect receive DidOpenTextDocumentParams");
                 Some(params.text_document.uri)
             }
-            DidSaveTextDocument::METHOD => {
```

Comment:
Curious why this is removed?

Reply:
This will cause unnecessary call of publishing diagnostics, which is already triggered by text change.